commcare_export/map_format.py

```diff
@@ -33,9 +34,21 @@ def parse_selected(value_expr, selected_expr_string):
     return Apply(Reference(SELECTED), value_expr, Literal(ref_val))
 
 
+def parse_template(value_expr, format_expr_string):
+    args_string = parse_function_arg(TEMPLATE, format_expr_string)
+    args = [arg.strip() for arg in args_string.split(',') if arg.strip()]
+    if len(args) < 1:
+        return Literal('Error: template function requires the format template: {}'.format(format_expr_string))
+    template = args.pop(0)
+    if not args:
+        args = [value_expr]
+    return Apply(Reference(TEMPLATE), Literal(template), *[Reference(arg) for arg in args])
```

Comment:
I know this parsing is very brittle, but no need for anything more robust at this stage